### PR TITLE
working nominatim 3.6. 

### PR DESCRIPTION
**Pull request by geohacker**
After several several several hours of debugging @batpad discovered that NOMINATIM_SETTINGS envvar must be set to the path of local.php. This is a cleaner setup and also has basic docs. Note that i've not added to docker-compose but you can use docker run and get everything running. 

This PR also adds a modified local.php that can pull envars dynamically and that helps with k8s deployment.

cc @Rub21 @batpad

### CODE REVIEW

**Comment by geohacker** [comment]
@Rub21 I recommend closing the `nominatim` branch and finishing your work on this branch.

**Comment by Rub21** [comment]
This is great, Thank you @batpad !!

File: images/nominatim/Dockerfile [hunk 1]
```diff
@@ -133,6 +133,7 @@ COPY init.sh /app/init.sh
 COPY start.sh /app/start.sh
 COPY startapache.sh /app/startapache.sh
 COPY startpostgres.sh /app/startpostgres.sh
+COPY local.php /app/src/build/settings/local_db.php
```
Comment (geohacker) [inline images/nominatim/Dockerfile]: Why is this needed @Rub21?

Comment (Rub21) [reply images/nominatim/Dockerfile]: @geohacker  according to the doc, we ned the configuration  to connect to the DB,   when you run your nominating locally you can have the local.php file in your local directory  and make a copy like here 👉  `https://github.com/mediagis/nominatim-docker/blob/master/3.6/startapache.sh#L2`  but when you run  in other environments such us Kubernetes you won’t  have the local.php in the cluster..  the idea is to have this local.php in the container, and when i run the container it will take that config file, in our case as `export NOMINATIM_SETTINGS=‘/app/src/build/settings/local_api.php’`

**Review by batpad — APPROVED** [review]

**Review by batpad — APPROVED** [review]
